### PR TITLE
DAT-2818 Infoboxes field mutators

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/DAT-2818

### CODE REVIEW

File: extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php [hunk 1]
```diff
@@ -145,7 +146,10 @@ protected function getSourceForChildren() {
 	protected function getValueWithDefault( \SimpleXMLElement $xmlNode ) {
 		$value = $this->extractDataFromSource( $xmlNode );
 		if ( !$value && $xmlNode->{self::DEFAULT_TAG_NAME} ) {
```
Comment (idradm) [inline extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php]: Maybe we could switch places for those two ifs, to make return statement less confusing

**Comment by idradm** [comment]
Could you add test for getSource testing for format tag?

File: extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php (hunk 1 above)
Comment (dianafa) [inline extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php]: added return statement

**Comment by idradm** [comment]
good :shipit:

**Pull request merged**